sources/Debugger/RegisterViewer.hpp

```diff
@@ -87,7 +87,11 @@ namespace ComSquare
 			//! @brief A 16 bits hexadecimal value.
 			SixteenBits,
 			//! @brief A 24 bits hexadecimal value.
-			TwentyFourBits
+			TwentyFourBits,
+			//! @brief Just like printf(%d, myVar);
+			Integer,
```

Comment:
What is the integer used for? Aren't hexadecimal values easier to read?

Reply:
Background number register is using an integer